package.json

```diff
@@ -3,38 +3,25 @@
   "version": "1.0.0",
   "description": "Leather wallet shared code",
   "scripts": {
-    "install:clean": "rm -rf \"**/node_modules\" && pnpm -r clean",
-    "install:fresh": "pnpm install:clean && pnpm i",
-    "install:nuke": "rm -rf pnpm-lock.yaml && pnpm install:fresh",
-    "lint:eslint": "eslint .",
-    "lint:eslint:fix": "pnpm lint:eslint --fix",
-    "lint:prettier": "prettier --check \"*.{ts,tsx,js,json}\"",
-    "lint:prettier:fix": "prettier --write \"*.{ts,tsx,js,json}\" *.js",
-    "lint:types": "tsc --noEmit",
-    "prepare": "husky install"
+    "prepare": "husky install",
+    "lint-projects:all": "nx run-many -t format,lint,typecheck --parallel=1",
```

Comment:
This is running each projects individual linting now right? Hence we don't need to install the linting packages at the top level

Reply:
Yeap! We might want to keep packages smaller to have more use out of nx caching in the future (e.g. like splitting up api folder into multiple api folders)